src/camera.py

```diff
@@ -186,78 +186,102 @@ def get_tilt_angle(self):
         return self._current_tilt_angle
 
 
+    def reset_camera(self):
+        """
+        Resets the camera to default position.
+
+        Sets the camera pan to 90 and the tilt to 0.
+        """
+        self._current_pan_angle = 90
+        self._current_tilt_angle = 0
+
+        self._pan_servo.write(np.clip(self._current_pan_angle
+                                      + self._pan_offset,
+                                      self.MIN_PAN_TILT_DEGREE,
+                                      self.MAX_PAN_DEGREE))
+        self._tilt_servo.write(np.clip(self._current_tilt_angle
+                                       + self._tilt_offset,
+                                       self.MIN_PAN_TILT_DEGREE,
+                                       self.MAX_TILT_DEGREE))
+
+
 def main():
     """
     Demonstrates the camera's panning and tilting functionality.
 
-    Creates a camera object and pans from 0 to 180, then from 180 to 0. The
-    camera then tilts from 0 to 150, then 150 to 0.
+    Creates a camera object using absolute degrees and pans from 0 to 180, then
+    from 180 to 0. The camera then tilts from 0 to 150, then 150 to 0.
+    Then creates a camera object using relavtive degrees and pans from 0 to 180,
+    then from 180 to 0. The camera then tilts from 0 to 150, then 150 to 0.
     """
     camera_delay = 0.05
+    camera_delay_long = 0.10
     picar.setup()
 
     # Creates a Camera object with absolute degrees (default).
     camera = Camera()
-
     print "Testing absolute camera degrees."
+
+    # Moving the camera to the right for panning.
+    camera.pan(0)
+    sleep(camera_delay_long)
```

Comment:
Is this where you were attempting to track time or something?

Reply:
No, when the camera requires larger movement the camera_delay time doesn't allow it enough time to travel the full distance